src/views/Survivor.vue

```diff
@@ -65,12 +77,26 @@ export default {
         return []
       },
       required: false
+    },
+    perksSHD: {
+      type: Object,
+      required: true
     }
   },
   data: function () {
+    let survivorsRaw = Object.keys(this.perksSHD.frames)
+    let survivors = []
+    // make sure array keys match the ids in file name. TODO maybe make sure no key is reassigned because of naming issues
+    for (let i = 0, sLen = survivorsRaw.length; i < sLen; i++) {
+      let s = survivorsRaw[i]
+      let k = Number(s.substr(0, 2))
```

Comment:
Please use more descriptive names for variables to prevent them to be missunderstood.